src/compiler/crystal/syntax/parser.cr

```diff
@@ -3344,6 +3347,10 @@ module Crystal
           index += 1
         end
 
+        if is_setter && args.size > 1
```

Comment:
Can't `is_setter` be replaced by `name.ends_with?('=')`?

Reply:
oh yeah! good catch